.github/workflows/lint.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         run: npm ci
 
       - name: Run linters 📑
-        uses: wearerequired/lint-action@v1.7.0
+        uses: wearerequired/lint-action@v1.8.0
         with:
           github_token: ${{ secrets.github_token }}
```

Comment:
```suggestion
        with:
```

No longer necessary

Reply:
Added in 80f7803ae91987893a2987165713b981ab5b99b0
Thanks! :wink: